spec/features/user_views_all_books_page_spec.rb

```diff
@@ -19,4 +19,19 @@
     expect(page).to have_css('.book', count: 2)
     expect(page).to have_css('.pagination')
   end
+
+  it 'should have genre of books' do
+    book1 = create :book
+    book2 = create :book
+
+    genre1 = create :genre, name: 'Genre1', books: [book1]
+    genre2 = create :genre, name: 'Genre2', books: [book2]
+
+    visit '/books'
+
+    expect(page).to have_css('.genre', text: 'Genre1')
+    expect(page).to have_css('.genre', text: 'Genre2')
+    expect(page).to have_link(genre1.name, genre_path(genre1))
+    expect(page).to have_link(genre2.name, genre_path(genre2))
```

Comment:
@Fatpaher There is no need to create two books for this spec. One is good enough.